ci/run-tests.ps1

```diff
@@ -59,6 +72,8 @@ $cmd_args_list = @( `
     "-nosplash", # No splash screen
     "-unattended", # Disable anything requiring user feedback
     "-nullRHI", # Hard to find documentation for, but seems to indicate that we want something akin to a headless (i.e. no UI / windowing) editor
+    "-stdout", # Print to output
+    "-ini:SpatialGDKSettings:[/Script/SpatialGDK.SpatialGDKSettings]:$additional_gdk_options" # Pass changes to configuration files from above
```

Comment:
I would move `[/Script/SpatialGDK.SpatialGDKSettings]:` to the if-block in the above options parsing, so that it's analogous to the else-block